CHANGELOG.md

```diff
@@ -5,21 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog], and this project adheres to
 [Semantic Versioning].
 
-## [0.6.2] - 2023-12-05
```

Comment:
Interesting, did you manually drop the dates from the release versions? or was that an auto-formatting? isn't against semantic versioning to included - because they seem useful.

Reply:
I removed them from the file for 2 reasons: (1) each is a guess based on when we think the actual release will be made (so it is possible to be wrong), and (2) they are redundant because github automatically adds the release date to the [releases page](https://github.com/MAAP-Project/gedi-subsetter/releases)

Reply:
But that would require a user to go dig into the Github releases and cross reference. It's much nicer to just have it in the log for humans to read.